src/Dropdown/Dropdown.js

```diff
@@ -412,14 +416,15 @@ class Dropdown extends Util.mixin(BindMixin) {
         onBlur={this.handleWrapperBlur}
         ref={this.setDropdownWrapperRef}
       >
-        <button
-          className={props.buttonClassName}
-          onClick={this.handleMenuToggle}
-          type="button"
-          disabled={props.disabled}
-        >
-          {this.getSelectedHtml(this.getSelectedID(), items)}
-        </button>
+        {React.createElement(
+          triggerTag,
+          {
+            className: props.buttonClassName,
+            onClick: this.handleMenuToggle,
+            disabled: props.disabled
+          },
+          this.getSelectedHtml(this.getSelectedID(), items)
```

Comment:
your implementation doesn't allow `triggerTag:"input"` because they arent allowed to have children 😢 
maybe we can change it to receive a ReactComponent instead of a string? 🤔 

IMHO like it is now, it is even worse then being bound to `button`